### DISCUSSION TITLE
VoWiFi not working after latest march update

### DISCUSSION THREAD

**Discussion by moshenoah188** [kyujin-cho/pixel-volte-patch#101] [Q&A]
Hi,

I have a pixel 7 pro and 2 days ago I updated the phone.
Since then the VoWiFi won't connect even that the option appear in the network options.
I tried rerunning the tool (ver 1.2.4) and add an ims APN but still no luck.

Can someone help fix this?

Phone ver is 13 (TQ2A.230305.008.C1).

Thanks,

**Comment by kyujin-cho** [comment]
Hi! Can you try https://github.com/kyujin-cho/pixel-volte-patch/issues/61#issuecomment-1443667286 and see if it works?

**Reply by moshenoah188** [reply]
Tried it, didn't work. :(

**Reply by moshenoah188** [reply]
Some updates to the problem:
- The IMS status is registered for VoLTE but as soon as I switch VoWiFi on, The IMS status become unregistered.
- I have put my sim in a different pixel phone using the tool and the VoWiFi connected and worked.

**Reply by kyujin-cho** [reply]
Latest update did really change something at VoWiFi, huh. Did you manage to enable VoWiFi with previous Pixel IMS (1.2.2) release?

**Reply by moshenoah188** [reply]
yes

**Reply by moshenoah188** [reply]
Also in the phone info screen (under *#*#4636#*#*) I see another phone index: 
![image](https://user-images.githubusercontent.com/126377936/228633986-d25adb1b-0e1a-4e17-a32f-d8b5fbf2173b.png)

is it ok?

**Reply by moshenoah188** [reply]
Hi,
@kyujin-cho you have Any solution or fix that I can attempt?
Thanks,

**Reply by kyujin-cho** [reply]
If you manage to enable VoLTE with previous version then please stick with that version for now. I'm keep looking for the reason why it stopped working on latest release.

**Reply by moshenoah188** [reply]
I tried using older version but it didn't worked.
I already update the phone to march update and the older version don't work.

**Reply by kyujin-cho** [reply]
So the March update could be the main reason. I'll take a look what's changed.
